ghost/core/core/server/services/members/service.js

```diff
@@ -193,6 +194,11 @@ module.exports = {
     ssr: null,
     verificationTrigger: null,
 
+    requestIntegrityTokenProvider: new RequestIntegrityTokenProvider({
+        themeSecret: settingsCache.get('theme_session_secret'),
+        tokenDuration: 1000 * 60 * 5
```

Comment:
Are we wanting to make this last a shorter duration than the magic links to check for spam activity? Those are 24h.

Reply:
I think so, these are generated immediately before use, so we could even have them be valid for as little as 60~ seconds. There should never be a reason for a session to use an old token.

Reply:
Got it, yeah. I was thinking it was verified w/ the magic link click, not on generating the magic link, for whatever reason. Sounds good to me.